.github/workflows/ci.yaml

```diff
@@ -14,21 +14,25 @@ jobs:
   build:
     env:
       COVER: true
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
 
     - name: Check out code
       uses: actions/checkout@v4
 
-    - name: Set up Go 1.21
+    - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        go-version: 1.21
+        # renovate: datasource=golang-version depName=golang
+        go-version: 1.21.6
       id: go
 
     - name: Get dependencies
+      env:
+        # renovate: datasource=github-tags depName=golangci/golangci-lint
+        GOLANGCI_LINT_VERSION: v1.55.2
```

Comment:
Keeps golangci-lint up to date. @tuunit do you still have the setup for the nightly stuff at hand to give this a quick test?

Reply:
We shouldn't update the go or go linter version automatically. That is really something we have to plan for. We should not manage the go and the golinter version through renovate.

There where sooo many breaking changes between v1.50 and v1.55